SupersonicSound/LowLevel/LowLevelSystem.cs

```diff
@@ -221,11 +221,19 @@ void IPreInitilizeLowLevelSystem.SetCallback(Action<LowLevelSystem, SystemCallba
         #endregion
 
         #region init/close
+        [Obsolete("Not sure why this method is in here, the system should already be initialized when passed in through the constructor")]
         public void Initialize(int maxchannels, LowLevelInitFlags flags)
```

Comment:
I see, but correct me if I'm wrong, but it seems that you can only initialize once, either the Studio, or the LowLevel system (and same goes for close/release)? So maybe we need to have a constructor in LowLevelSystem that is used by the Studio and a different one when LowLevel is used standalone, and then use that to determine if these should be called or not?